spec/models/link_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Link, type: :model do
   it "determines the font awesome character to use from the url" do
```

Comment:
Could you mark this spec as pending for now, the link feature is not being used yet.(Use `xit`)

Reply:
will do